locales.software/1038-hu.cfos.txt

```diff
@@ -763,7 +763,7 @@ err_os_minxp = "Az ön operációs rendszere %?.\n\n%? futtatásához Windows XP
 err_os_sup = "%p hivatalosan csak a %? vagy magasabb verziót támogatja.\n\n"
              "Lehetséges, de általunk nincs tesztelve, hogy a %p elfut-e az ön %? operációs rendszerén.\n\n"
              "Biztos szeretné folytatni a telepítést?";
-err_uninst_driver = "Az eszközmeghajtó eltávolítása sikertelen.";
+err_uninst_driver = "Az eszközmeghajtó (driver) eltávolítása sikertelen.";
```

Comment:
pnet_start_info = "Ajánlott, hogy %p automatikusan elinduljon bejelentkezéskor. "
"Ha csak ritkán szeretné nyílvánosan megosztani adatait, akkor kézzel indítsa el a %p-t.";

The other texts seem OK now. 
I left the "Az eszközmeghajtó (driver) eltávolítása sikertelen."; as it is. it's clear it is a duplication, but I think it is good for clarification. In previous  texts "driver" and "eszközmeghajtó" also were used simultaniously. I think "driver" alone is a bit confusing as it has at least two meanings in English (in computer environment).